index.js

```diff
@@ -282,6 +283,12 @@ function runCmd_ (cmd, pkg, env, wd, opts, stage, unsafe, uid, gid, cb_) {
     }
     procError(er)
   })
+  byline(proc.stdout).on('data', function (data) {
```

Comment:
In the interests of not making a change that we'll need to then break soon after…

What we've been intending to do with npm is to buffer the output from lifecycle scripts and…

1. If there's an error, output at warn level.
2. If there's not an error, output at suppressed by default level (possibly lower than verbose…)

I suppose we _can_ support that later by adding another option that provides a function that gets the proc object and can do with stdout/stderr as it likes (as an alternative to this).